src/home/search.vue

```diff
@@ -80,11 +77,10 @@
     button.extra.circle.fill(data-ui="#search4")
       i search
       menu#search4.max
```

Comment:
When clicking the close and arrow_back we keep the focus maybe we should remove the focus ?
![image](https://github.com/beercss/beercss/assets/43939417/a3f24489-745e-4bef-9d1c-ce6c0053caca)
![image](https://github.com/beercss/beercss/assets/43939417/9d52e982-ef3c-41c8-a3b5-ed8eb29782af)

Reply:
This can be another PR too, like the other changes to do in our search.